src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
@markelog Since this PR contains the revert anyway and it makes sense to land it as 2 commits, maybe we should just merge the revert now (& fix the build this way) and then just review the proper patch? In this way we also wouldn't be in a hurry to quickly fix the build as it will be fixed already.

Now we're commenting on a Frankenstein diff...